python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -384,6 +387,35 @@ class CalibrateImageConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Cali
         doc="If True, include astrometric errors in the output catalog.",
     )
 
+    run_sattle = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="If True, sattle service will populate catalog for use in "
+            "ip_diffim.detectAndMeasure alert verification."
+    )
+
+    sattle_historical = sattle_host = pexConfig.Field(
```

Comment:
I assume this is a typo:
```suggestion
    sattle_historical = pexConfig.Field(
```